.github/actions/build-windows-artifacts/action.yml

```diff
@@ -76,5 +76,7 @@ runs:
       uses: ./.github/actions/upload-artifacts
       with:
         artifacts-dir: ${{ inputs.artifacts-dir }}
-        target-file: target/${{ inputs.arch }}/${{ inputs.cargo-profile }}/greptime
+        target-file: |
+          target/${{ inputs.arch }}/${{ inputs.cargo-profile }}/greptime
+          target/${{ inputs.arch }}/${{ inputs.cargo-profile }}/greptime.pdb
```

Comment:
@zyy17 Does this ensure we will release the pdb file? I remember that we only upload the greptime binary in the pipeline.

Reply:
You are right. We have to support to pack multiple files in upload artifacts action.